jbmc/regression/jbmc/array-clone/ArrayClone.java

```diff
@@ -0,0 +1,65 @@
+public class ArrayClone {
+  public static void cloneReferenceArray() {
+    ArrayClone[] a = {new ArrayClone(), new ArrayClone() };
+    ArrayClone[] clone = a.clone();
+
+    assert(a.length == clone.length);
```

Comment:
Just comparing the length is a rather weak test, but better than nothing I guess! :-)

Reply:
🚫 Given this has no descriptions, I'd expect a comparison of arrays (array1 != array2) and references of objects/values of primitives. As it stands these tests are rather weak.